admin/sql/create_tables.sql

```diff
@@ -104,4 +104,26 @@ CREATE TABLE follow_list (
 );
 ALTER TABLE follow_list ADD CONSTRAINT follow_list_name_creator_key UNIQUE (name, creator);
 
+CREATE TABLE recommendation.cf_recording (
+  id                  SERIAL, -- PK
+  user_id             INTEGER NOT NULL, --FK to "user".id
+  msid                UUID NOT NULL,
+  type                recording_type,
+  created             TIMESTAMP WITH TIME ZONE DEFAULT NOW()
+);
+
+CREATE TABLE recommendation.recommender (
+  id                  SERIAL, --PK
```

Comment:
recommenders don't have to be made by existing MB user? Does it make sense for us to link them here so that we can group better (e.g. list all of someone's recommenders)

Reply:
Are you suggesting that we link to MB accounts, rather than having random emails, etc? (just trying to understand your comment). Sounds good though.

Reply:
Recommenders of existing MB users sounds like a good idea.

Reply:
OK, then this should be a FK to the "user" table, which makes a lot more sense. Also we can remove the author_email then.